web/styles/inbox.css

```diff
@@ -225,14 +225,17 @@
             }
 
             .collapsible-button {
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                width: 1.5em; /* 24px looked good at 16px */
+
                 &:hover {
                     cursor: pointer;
                 }
 
                 .zulip-icon-arrow-down {
-                    font-size: 16px;
-                    padding: 5px 4px;
```

Comment:
Is there a reason these padding and margin values have been removed? I wonder if those aren't to blame for the leftward shifts in the screenshots?

Reply:
Yeah I switched the centering them with flex, but then I had to make the parent a larger width. That is why there's the leftward shift, but I didn't put that much effort into making it match what it was before until I had confirmation that we actually still want that (since it looks a bit strange to me) since it was getting late and I was heading to bed.

Reply:
Yeah, it is strange, but I'd err on the side of introducing as minor a change as possible. What's needed on all the inbox rows is a gridded rewrite, so I don't think introducing flexbox here is the right move right now.

Reply:
Ah okay, I'll keep the padding positioning then and convert the values all to em.

Reply:
Done and updated screenshots! (there's now a visible change in the buddy list but that's because of #33428 being merged, and it didn't feel worth redoing the before shots)

Reply:
Awesome. Thanks, Evy!